src/app/_components/Avatar.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import Image from 'next/image'
+
+import type { ImageProps } from '@/types/sharedProps/imageType'
+
+type ImagePropsWithoutAlt = Omit<ImageProps, 'alt'>
+
+export type Author = {
+  firstName: string
+  lastName: string
+  image?: ImagePropsWithoutAlt
+}
+
+export function Avatar({ authors }: { authors: Array<Author> }) {
+  const imageSize = 32
+
+  if (!authors || authors.length === 0) {
+    throw new Error('Avatar component requires at least one author')
+  }
+
+  const authorsNames = authors
+    .map((author) => `${author.firstName} ${author.lastName}`)
+    .join(' & ')
+
+  const renderContent = ({ firstName, lastName, image }: Author) => {
```

Comment:
I'll let @CharlyMartin review this properly first but I wanted to drop in an early review. Let's refactor this using Clean Code JS principles.

Also - Can we start making the `Card` component more modular? We'll be doing a refactor soon - however, we can already have something like `Card.Avatar`.